### DISCUSSION TITLE
How do I open program with arguments via shell open?

### DISCUSSION THREAD

**Discussion by luvor** [tauri-apps/tauri#6080] [Q&A]
As in cmd like this: start "" "C:\Program Files\Google\Chrome\Application\chrome.exe" -incognito

I tried via rust 
```
use std::process::Command;
Command::new("start").arg("\"\" \"C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe\" -incognito")
```
But it raises and error that program not found (I tried change to split it to args like this agrs(["\"\"", "C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe", "-incognito"], but it also didn't worked)

**Comment by FabianLars** [comment]
The shell.open api indeed does not (and probably won't ever?) support arguments.

Did you also try the Command like this?
```rs
use std::process::Command;
Command::new("C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe").arg("-incognito")
```

`start` is not a standalone executable, and only a script inside cmd.exe / powershell, meaning that you'd have to do something like this (i didn't test it, but it should be something in that direction, i still recommend the approach above though)
```rs
use std::process::Command;
Command::new("cmd.exe").args(["start", "/C", "\"path to chrome -incognito\""])
```
